ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +407,173 @@ def test_trustdomain_disable_disables_subdomain(self):
+            with open(os.path.join(
+                    tmpdir, 'ssh_priv_{}'.format(i)), 'w') as fp:
+                fp.write(ssh_key_pair[0])
+
+        keys_opts = ' '.join(['--ssh "{}"'.format(k) for k in pub_keys])
+        cmd = 'ipa user-mod {} {}'.format(username, keys_opts)
+        self.master.run_command(cmd)
+
+        # connect with first SSH key
+        user_ssh_priv_key = os.path.join(tmpdir, 'ssh_priv_1')
+
+        # change private key permission to comply with SS rules
+        os.chmod(user_ssh_priv_key, 0o600)
+
+        # add group parent
+        cmd_output = master.run_command(['ipa', 'group-add', 'parent'])
+        assert 'Added group "parent"' in cmd_output.stdout_text
+
+        # add group child1
+        cmd_output = master.run_command(['ipa', 'group-add', 'child1'])
+        assert 'Added group "child1"' in cmd_output.stdout_text
+
+        # add group child2
+        cmd_output = master.run_command(['ipa', 'group-add', 'child2'])
+        assert 'Added group "child2"' in cmd_output.stdout_text
+
+        # add group members
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'child1', '--groups', 'parent'])
+        assert 'Group name: child1' in cmd_output.stdout_text
+        assert 'Member groups: parent' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'child2', '--groups', 'child1'])
+        assert 'Group name: child2' in cmd_output.stdout_text
+        assert 'Member groups: child1' in cmd_output.stdout_text
+        assert 'Indirect Member groups: parent' in cmd_output.stdout_text
+
+        # add user to group 'parent'
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'parent', '--users', username])
+        assert 'Group name: parent' in cmd_output.stdout_text
+        assert 'Member users: {}'.format(username) in cmd_output.stdout_text
+        assert 'Member of groups: child1' in cmd_output.stdout_text
+        assert 'Indirect Member of group: child2' in cmd_output.stdout_text
+
+        # clear sssd_cache
+        clear_sssd_cache(master)
+
+        # user lookup
+        cmd_output = master.run_command(['ipa', 'user-show', username])
+        assert 'parent' in cmd_output.stdout_text
+        assert 'ipausers' in cmd_output.stdout_text
+        assert 'child1' in cmd_output.stdout_text
+        assert 'child2' in cmd_output.stdout_text
+
+        clear_sssd_cache(master)
+
+        sshcon = paramiko.SSHClient()
+        sshcon.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+
+        try:
+            cmd = 'ssh -i {} -q {}@{} groups'.format(user_ssh_priv_key,
+                                                     username, client.hostname)
+            cmd_output = master.run_command(cmd)
```

Comment:
The command is run on the master but the file user_ssh_priv_key was generated on the controller and never copied to the master. This is why you get `Warning: Identity file /tmp/tmp-9ywpcf6o/ssh_priv_1 not accessible: No such file or directory.`

Reply:
fixed it